packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -0,0 +1,276 @@
+import { BigNumber } from 'bignumber.js'
+import * as React from 'react'
+import { Clipboard, StyleSheet, View } from 'react-native'
+import css from 'src/dev/ValidatorsList.scss'
+import { H1 } from 'src/fonts/Fonts'
+import { I18nProps, withNamespaces } from 'src/i18n'
+import Chevron, { Direction } from 'src/icons/chevron'
+import { HEADER_HEIGHT } from 'src/shared/Styles'
+import { colors, standardStyles, textStyles } from 'src/styles'
+
+interface ValidatorsListProps {
+  data: any
+  isLoading: boolean
+}
+
+export interface State {
+  expanded?: number
+}
+
+class ValidatorsListApp extends React.PureComponent<ValidatorsListProps & I18nProps, State> {
+  state = {
+    expanded: undefined,
+  }
+
+  cutAddress(address: string) {
```

Comment:
I typically just keep helper functions that don't operate on state/props outside of the component